v3/src/components/axis/hooks/use-sub-axis.ts

```diff
@@ -96,14 +98,37 @@ export const useSubAxis = ({
         renderNumericAxis = () => {
           select(subAxisElt).selectAll('*').remove()
           const numericScale = d3Scale as unknown as ScaleLinear<number, number>
-          // When displaying bars, set the domain to [0, 100], otherwise use the default.
           if (graphModel.pointDisplayType === "bars") {
+            // When displaying bars, set the domain to [0, 100]. TODO: Fix this. The domain does need to be changed for
+            // bars, but it should be based on the data, not always the same static values.
```

Comment:
There is a [separate story](https://www.pivotaltracker.com/story/show/187142716) for this.